cmd/bosun/sched/check.go

```diff
@@ -125,8 +126,18 @@ func (s *Schedule) runHistory(r *RunHistory, ak models.AlertKey, event *models.E
 		// save unless incident is new and closed (log alert)
 		if incident != nil && (incident.Id != 0 || incident.Open) {
 			_, err = data.UpdateIncidentState(incident)
+			if err != nil {
+				return
+			}
+			err = data.SetRenderedTemplates(incident.Id, rt)
+			if err != nil {
+				return
+			}
 		} else {
 			err = data.SetUnevaluated(ak, event.Unevaluated) // if nothing to save, at least store the unevaluated state
+			if err != nil {
```

Comment:
This error checking block is a noop if there is no code following. Same thing directly above this block too.